connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinatorTest.java

```diff
@@ -105,7 +106,7 @@
     @Parameters
     public static Iterable<?> mode() {
         return Arrays.asList(new Object[][]{
-                {ConnectProtocolCompatibility.EAGER, 1},
+                {EAGER, 1},
                 {ConnectProtocolCompatibility.COMPATIBLE, 2}});
```

Comment:
If you change the line above, then let's change this one too and fix the asymmetry